src/constants/tokens.ts

```diff
@@ -536,8 +548,11 @@ export const eligibleLeveragedExchangeIssuanceTokens = [
   Matic2xFLIP,
 ]
 
+// TODO: create beta/staging list? (env === staging)
```

Comment:
I think suggestion would work better as an issue that could be discussed.

Reply:
For sure 👌 For now I was just going to remove these tokens manually before pushing to production. Once it get's annoying I'd think about a solution here. I'm going to create a ticket on pivotal.